src/encoded/schemas/biosample_characterization.json

```diff
@@ -26,6 +26,15 @@
         "schema_version": {
             "default": "11"
         },
+        "antibodies": {
```

Comment:
Why does it have to be an array? why one biosample characterization will have multiple antibodies?

Reply:
In principle, a blot can have multiple antibodies marking different proteins...